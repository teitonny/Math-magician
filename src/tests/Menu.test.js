import React from 'react';
import Renderer from 'react-test-renderer';
import Menu from '../components/menu';

it('renders the Menu correctly', () => {
  const tree = Renderer.create(<Menu />).toJSON();
  expect(tree).toMatchSnapshot();
});
