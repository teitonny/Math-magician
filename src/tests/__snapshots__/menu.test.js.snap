// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the Menu correctly 1`] = `
<div
  className="navContainer"
>
  <div
    className="mathLogo"
  >
    <a
      href="/"
      onClick={[Function]}
    >
      Math magician
    </a>
  </div>
  <ul
    className="navbar"
  >
    <li
      className="nav-item"
    >
      <a
        href="/"
        onClick={[Function]}
      >
        Home
      </a>
    </li>
    <li
      className="nav-item"
    >
      <a
        href="/Calculator"
        onClick={[Function]}
      >
        Calculator
      </a>
    </li>
    <li
      className="nav-item"
    >
      <a
        href="/Quote"
        onClick={[Function]}
      >
        Quote
      </a>
    </li>
  </ul>
</div>
`;
